ide/csl.api/test/unit/src/org/netbeans/modules/csl/api/test/CslTestBase.java

```diff
@@ -4384,8 +4384,11 @@ protected ComputedHints computeHints(final NbTestCase test, final Rule hint, Str
                             }
                         }
                         if (HintsSettings.getSeverity(manager, ucr) == HintSeverity.CURRENT_LINE_WARNING) {
-                            manager.setTestingRules(null, Collections.EMPTY_MAP, testHints, null);
+                            manager.setTestingRules(null, testHints, testHints, null);
                             provider.computeSuggestions(manager, context, hints, caretOffset);
+                        } else if(HintsSettings.getSeverity(manager, ucr) == HintSeverity.ERROR || HintsSettings.getSeverity(manager, ucr) == HintSeverity.WARNING) {
+                            manager.setTestingRules(null, testHints, testHints, null);
+                            provider.computeErrors(manager, context, hints, new ArrayList<Error>());
```

Comment:
Did you test, that all current tests invoking `computeHints`, potentially transitively, still work?

Reply:
No, not yet. I just thought that the pipes will show me the result and I just saw some failing tests of Latte. PHP where not triggert here at github inside my branch. I will just revert this change as you said and will try to implement a better logic into the HtmlHintsProvider as similiar as for PHP and JS.